content/oc/_index.md

```diff
@@ -162,4 +162,79 @@ done
 
 ```
 oc logs <pod> --all-containers
-```
\ No newline at end of file
+```
+
+# Create Kubeconfig out of credentials / Merge multiple Kubeconfigs
+
+For creating a Kubeconfig file from a given credentials we can run the following commands:
+
+```sh
```

Comment:
Please remove the `sh` it makes the render weird (see the preview https://deploy-preview-15--openshifttips.netlify.com/oc/#create-kubeconfig-out-of-credentials-merge-multiple-kubeconfigs )